client/src/store/metadata/metadata.defaults.ts

```diff
@@ -175,7 +175,8 @@ export const defaultModelFields: ModelFields = {
         selected: true,
         subtitleOption: eSubtitleOption.eNone,
         id: 0
-    }]
+    }],
+    skipSceneGenerate: true
```

Comment:
I'm not sure about the impact of this default being set to `true`, but the default behavior is for us to generate scenes, not skip scene generation.  Perhaps this should be updated to `false`?